cooltools/cli/eigs_cis.py

```diff
@@ -171,7 +171,6 @@ def eigs_cis(
         clr_weight_name=clr_weight_name,
         ignore_diags=ignore_diags,
         clip_percentile=99.9,
-        sort_metric=None,
     )
```

Comment:
is this desired? this issue suggests to expose `sort_metric` https://github.com/open2c/cooltools/issues/311

Reply:
I removed it to be replaced with `corr_metric` (used for both flipping and reordering) and `reorder` instead.

Reply:
aha, gotcha-- thoughts on `phasing_metric` or `phasing_corr_metric` to help remind the user what this is for?